sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -142,13 +142,25 @@ case class RowDataSourceScanExec(
       handledFilters
     }
 
+    val optionOutputMap =
+      if (pushedDownOperators.limit.isDefined && pushedDownOperators.sortValues.nonEmpty) {
+        val pushedTopN =
+          s"""
+             |ORDER BY ${seqToString(pushedDownOperators.sortValues.map(_.describe()))}
+             |LIMIT ${pushedDownOperators.limit.get}
```

Comment:
nit: shall we output only one line? otherwise the plan EXPLAIN result may look very weird.

Reply:
OK